test/jdk/sun/security/pkcs11/Config/ReadConfInUTF16Env.java

```diff
@@ -21,17 +21,36 @@
  * questions.
  */
 
+/* @test
+ * @bug 8187023
```

Comment:
Should it have the enhancement bug id too.

Reply:
My understanding is that we should only include the id of product bugs, no test enhancements. That was a comment somebody added in a previous PR. The reason is that you can track changes in the test files with git history, whereas it is only possible to track the related product bug id by adding it manually to the class header